client/reader/list-stream/style.scss

```diff
@@ -124,10 +135,6 @@
 .list-stream__header-edit {
 	margin-left: auto;
 
-	.list-stream__header-action-label {
```

Comment:
This is no good - it'll be hidden from screen readers too. I've used the `screen-reader-text` class instead.